extensions/project-descriptor.md

```diff
@@ -188,6 +190,19 @@ Either a `version`, `uri`, or `script` table MUST be included, but MUST NOT incl
 
 The `api` and `inline` key MUST be defined in the `script` table. The value of the `inline` key will be used as the build script for the [inline buildpack](#Definitions) this entry represents. The value of the `api` key defines its Buildpack API compatibility, and the `shell` key defines the shell used to execute the `inline` script.
 
+#### `io.buildpacks.pre.group` (optional)
+
+This table MAY contain a list of buildpacks to insert at the beginning of an automatically detected group. Given an order with multiple groups, the list of `pre` buildpacks will be inserted at the beginning of each automatically detected group such that they are run as if they were originally included in the group. Each phase of the injected buildpack(s) will execute as normal.
```

Comment:
I'm curious about the use of the term "detected" here. It's my understanding that pre and post buildpacks would _always_ execute for every declared group. This phrasing makes it sounds like the pre and post buildpacks would only execute after the initial declared groups have gone through detection and a certain group was selected.

Reply:
This seems to be a replay of [this comment](https://github.com/buildpacks/spec/pull/233#discussion_r662515350) which doesn't seem to have been addressed.

Reply:
@jromero does this make it clearer?

```suggestion
This table MAY contain a list of buildpacks to insert at the beginning of every group. Given an order with multiple groups, the list of `pre` buildpacks will be inserted at the beginning of each group such that they are run as if they were originally included in the group. Each phase of the injected buildpack(s) will execute as normal.
```

Reply:
I agree with @jromero 's concern and prefer @hone 's suggestion to the current wording.